runtime/onert/core/include/ir/OpSequence.h

```diff
@@ -58,8 +58,11 @@ class OpSequence
   const OperandIndexSequence &getOutputs() const { return _outputs; }
   void setInputs(const OperandIndexSequence &indexes) { _inputs = indexes; }
   void setOutputs(const OperandIndexSequence &indexes) { _outputs = indexes; }
-  void replaceInput(const OperandIndex &from, const OperandIndex &to) { _inputs.replace(from, to); }
-  void replaceOutput(const OperandIndex &from, const OperandIndex &to)
+  void replaceInputs(const OperandIndex &from, const OperandIndex &to)
```

Comment:
`replaceInput` is right? the from seems an index not indice.

Reply:
It replaces the same inputs as `from` with `to`

Reply:
It means the name. An INDEX, Not INDICES. Right? If so, I think `replaceInput` is right.

Reply:
Your meaning is to replace `_inputs`?

Reply:
I thought an input of _inputs

Reply:
If your intuition is so, you're right.

Reply:
It means the name. An INDEX, Not INDICES. Right? If so, I think `replaceInput` is right. However, if your meaning is to replace `_inputs`? If your intuition is so, you're right.
(I thought replacing an input of `_inputs`.)

Reply:
This function may replace inputs among `_inputs`, but not an input among `_inputs`.
If `_inputs` has 0 input as `from`, this function does not replace.
If `_inputs` has 1 input as `from`, this function replaces an input of `_inputs` with `to`.
If `_inputs` has 2 or more of the same inputs as `from`, this function replaces the same inputs with `to`.